iban/iban.go

```diff
@@ -91,6 +91,16 @@ func Validate(value string) error {
 	return err
 }
 
+// ValidateWithLength validates iban code and returns its total length.
+// It ignores any garbage after the IBAN code.
+func ValidateWithLength(value string) (int, error) {
+	struc, err := validateIgnoringTrailingGarbage(value)
+	if err != nil {
+		return 0, err
+	}
+	return 4 + struc.Length(), nil
```

Comment:
@dkegel-fastly maybe a comment explaining why to add 4? Or maybe making it a named constant would clarify that? I see 4 is also used below too.